modules/genomic_browser/php/gwasbrowser.class.inc

```diff
@@ -53,10 +52,7 @@ class GwasBrowser extends \NDB_Page implements ETagCalculator
      */
     private function _handleGET(ServerRequestInterface $request) : ResponseInterface
     {
-        $filter      = new HasAnyPermissionOrUserSiteMatch(
```

Comment:
This code was removed because no site information is available from [the query](https://github.com/aces/Loris/blob/3656cc12f6b7b9ff10de457627d48f1059112b74/modules/genomic_browser/php/provisioners/gwasprovisioner.class.inc#L27).